pom.xml

```diff
@@ -271,11 +271,6 @@
     </dependencies>
     <dependencyManagement>
         <dependencies>
-            <dependency>
-                <groupId>org.jenkins-ci.plugins</groupId>
-                <artifactId>apache-httpcomponents-client-4-api</artifactId>
```

Comment:
So, IIUC, if not for the bug in `AbstractHpiMojo `, this would have been correct? And the exclusion for  `org.apache.httpcomponents:httpclient` from `git-server` was due to an outdated `git-client` dep at some point and could be deleted if not for the bug? Or are `exclusions` in a POM specific only to that POM and not to upstream POMs, so we have to reapply them here?

The exclusion of `apache-httpcomponents-client-4-api` doesn't matter at runtime because it's actually going to come in as a plugin through the `git-server` to `git-client` dependency chain, right? This is just a hack to fix the packaging here?

Reply:
If the mojo were fixed, I _think_ this could have been kept and the `exclusion` removed.

Exclusions are heritable.

Reply:
> doesn't matter at runtime

Right.